### DISCUSSION TITLE
Ellipsoid

### DISCUSSION THREAD

**Discussion by mikhailpaltsev** [OpenGATE/opengate#641] [Technical issues] [answered]
When ellipsoid volume will be available? Or if there is, how do i do it?

**Comment by tbaudier** [comment, marked as answer]
Hello @mikhailpaltsev 

For the moment Ellipsoid volume is not available in Gate10. But you can try to add it and propose a pull request.
To start you can follow the developper guide here: https://opengate-python.readthedocs.io/en/master/developer_guide/developer_guide_installation.html

I think you can follow the example of G4Tubs to add the G4Ellipsoid class: https://github.com/OpenGATE/opengate/blob/master/core/opengate_core/g4_bindings/pyG4Tubs.cpp

**Comment by tbaudier** [comment]
It was integrated with that PR: https://github.com/OpenGATE/opengate/pull/713
